checks/check_extra7147

```diff
@@ -25,27 +25,32 @@ CHECK_CAF_EPIC_extra7147='Data Protection'
 
 extra7147(){
   for regx in $REGIONS; do
-  LIST_OF_VAULTS=$($AWSCLI glacier list-vaults $PROFILE_OPT --region $regx --account-id $ACCOUNT_NUM --query VaultList[*].VaultName --output text 2>&1|xargs -n1)
-  if [[ $(echo "$LIST_OF_VAULTS" | grep -E 'AccessDenied|UnauthorizedOperation|AuthorizationError') ]]; then
-      textInfo "$regx: Access Denied trying to list vaults" "$regx"
+  LIST_OF_VAULTS=$($AWSCLI glacier list-vaults ${PROFILE_OPT} --region "${regx}" --account-id "${ACCOUNT_NUM}" --query VaultList[*].VaultName --output text 2>&1|xargs -n1)
+  if grep -q -E 'AccessDenied|UnauthorizedOperation|AuthorizationError' <<< "$LIST_OF_VAULTS"; then
+      textInfo "$regx: Access Denied trying to list vaults" "${regx}"
+      continue
+  fi
+  # Check for unsupported regions
+  if grep -q -E 'ThrottlingException' <<< "$LIST_OF_VAULTS"; then
```

Comment:
This is not actually Throttling issue, it is not supported in that region. It may confuse the user despite the API gives that error.

Reply:
Yes, you're right but the error thrown by AWS CLI is "ThrottlingException".